goblint.opam

```diff
@@ -17,9 +17,8 @@ depends: [
   "dune"
   "zarith_stubs_js"
   "goblint-cil"
-  "batteries"
-  "qcheck"
-  # "z3"
```

Comment:
This Z3 comment should stay. Although I guess it can be annoying to keep around if the opam file is autogenerated.

Reply:
Oops! Yeah, I generated the opam file with `dune build @install` and removed that line accidentally.